doc/api/os.md

```diff
@@ -33,205 +43,282 @@ defined are described in [OS Constants][].
 added: v0.3.3
 -->
 
-Returns an array of objects containing information about each CPU/core
-installed: model, speed (in MHz), and times (an object containing the number of
-milliseconds the CPU/core spent in: user, nice, sys, idle, and irq).
+The `os.cpus()` method returns an array of objects containing information about
+each CPU/core installed.
 
-Example inspection of os.cpus:
+The properties included on each object include:
+
+* `model` {String}
+* `speed` {number} (in MHz)
+* `times` {Object}
+  * `user` {number} A number of milliseconds the CPU has spent in user.
+  * `nice` {number} A number of milliseconds the CPU has spent in nice.
+  * `sys` {number} A number of milliseconds the CPU has spent in sys.
+  * `idle` {number} A number of milliseconds the CPU has spent in idle.
+  * `irq` {number} A number of milliseconds the CPU has spent in irq.
```

Comment:
btw, from `proc(5)` for Linux:

```
user   (1) Time spent in user mode.

nice   (2) Time spent in user mode with low priority (nice).

system (3) Time spent in system mode.

idle   (4) Time spent in the idle task.  This value should be USER_HZ times the second entry in the /proc/uptime pseudo-file.

iowait (since Linux 2.5.41)
      (5) Time waiting for I/O to complete.

irq (since Linux 2.6.0-test4)
      (6) Time servicing interrupts.

softirq (since Linux 2.6.0-test4)
      (7) Time servicing softirqs.

[…]
```

I’d expect these values to mean similar things on other OS, so it might be worth using descriptions based on these?